packages/ui-extensions/src/surfaces/admin/components/Divider/Divider.ts

```diff
@@ -4,9 +4,9 @@ export interface DividerProps {
   /**
    * Specify the direction of the divider.
    *
-   * @defaultValue 'horizontal'
+   * @defaultValue 'block'
```

Comment:
This should be `inline`

Reply:
It's `inline`. If you check the implementation logic on the component, the default behaviour is `inline`.

Reply:
```suggestion
   * @defaultValue 'inline'
```

Reply:
Fixed in e6b5b62622858f392e8fc6f53117cfc59f16ebff